web_server/html/_i18n/Bahasa Indonesia.yml

```diff
@@ -0,0 +1,288 @@
+#  _ _ _ _     _____ _ _ _ _____ 

```

Comment:
Rename this to `indonesia.yml` and place it in https://github.com/davinhanif/Wi-PWN/blob/78d0cfa76c8b117e4deb0dd894d7e804020a1675/web_server/html/_config.yml